### DISCUSSION TITLE
CUD operation is possible on the Pimcore Enterprise PXP page.

### DISCUSSION THREAD

**Discussion by arulselvan008** [pimcore/pimcore#15092] [Q&A / Support]
Hi,

I inquired about the possibility of adding CUD (Create, Update, Delete) functionality to the data object in PXP (Pimcore Experience Portal) page in my project. Our management has requested that we implement the ability to add and edit products directly from the PXP page.

As you are aware, the PXP page is an important component of our project, and we would greatly benefit from the ability to perform CUD operations from within it. I have searched for information on how to add this functionality, but have been unable to find any documentation or resources on the topic.

I would appreciate any guidance or assistance you can provide in implementing CRUD functionality on the PXP page. If there are any existing resources or documentation available, please let me know where I can find them.

**Comment by arulselvan008** [comment]
anybody Can you help us here?

**Reply by rmpimconaut** [reply]
The demand for extended PXP functionality and especially the requirement for uploading product data and in addition the editing and enhancing via frontend UI is a comprehensible one. We already received similar requests and have those topics in our pipeline. But it is not that easy because this step has several impacts to the single-source-of-data principle. It affects data quality management, workflow processes, version tracking, data modelling and many more. To keep a long story short: there is no out of the box bundle/add-on which offers these functionalities. But be assured that a reasonable solution is in planning.
